src/features/errors/pages/AsyncErrorBoundary.tsx

```diff
@@ -31,12 +32,19 @@ export const AsyncErrorBoundaryWithoutNavigation = ({
   const { reset } = useQueryErrorResetBoundary()
 
   useEffect(() => {
-    if (
-      // we already captures MonitoringError exceptions (in AsyncError constructor)
-      !(error instanceof MonitoringError) ||
-      // we don't need to capture those errors
-      !(error instanceof ScreenError)
-    ) {
+    const shouldNotCapturedApiError = Boolean(
+      error instanceof ApiError && !isApiCapturedException(error.statusCode)
+    )
+    // we already captures MonitoringError exceptions (in AsyncError constructor)
+    // we don't need to capture those errors
+    // we don't to capture API errors 5xx
```

Comment:
Il manque un mot entre `don't` et `to` non ?

Reply:
J'ai un `to` qui a ripé ^^